src/wp-includes/class-wp-styles.php

```diff
@@ -165,14 +165,11 @@ public function do_item( $handle, $group = false ) {
 			$ver = $ver ? $ver . '&amp;' . $this->args[ $handle ] : $this->args[ $handle ];
 		}
 
-		$src                   = $obj->src;
-		$ie_conditional_prefix = '';
-		$ie_conditional_suffix = '';
-		$conditional           = isset( $obj->extra['conditional'] ) ? $obj->extra['conditional'] : '';
-
+		$src         = $obj->src;
+		$conditional = isset( $obj->extra['conditional'] ) ? true : false;
```

Comment:
Same, this potentially treats falsy conditionals as conditionals where before they were ignored:

```suggestion
		$conditional = (bool) ( $obj->extra['conditional'] ?? false );
```

And I think it could be moved earlier right after `$obj` is declared.